flocker/node/test/test_docker.py

```diff
@@ -172,10 +190,10 @@ def got_list(units):
 
         def test_container_name(self):
             """
-            Each unit also records the name of the container it is running in.
+            Each container also records the container name twice.
```

Comment:
Hehe. I don't think this is an improvement. 
Maybe just link to https://clusterhq.atlassian.net/browse/FLOC-819 with a note that container_name and name are always the same and this test should probably be removed.